server/routers/viewer.tsx

```diff
@@ -135,6 +145,14 @@ const loggedInViewerRouter = createProtectedRouter()
               team: null,
             },
             select: eventTypeSelect,
+            orderBy: [
+              {
+                position: "desc",
+              },
+              {
+                id: "asc",
+              },
+            ],
```

Comment:
a bit ugly to have this repeated - would be nicer to have it as function or a way to set default sorting in prisma